3d_armor/api.lua

```diff
@@ -117,6 +119,14 @@ armor.register_armor = function(self, name, def)
 		end
 		return armor:equip(player, itemstack)
 	end
+	-- The below is a very basic check to try and see if a material name exists as part
+	-- of the item name. However this check is very simple and just checks theres "_something"
+	-- at the end of the item name and logging an error to debug if not.
+	local check_mat_exists = string.match(name, "%:.+_(.+)$")
+	if check_mat_exists == nil then
+		minetest.debug("WARNING:3d_armor - Registered Armor "..name..
+		" dosen't have \"_material\" specified at the end of the item registration name")
```

Comment:
```suggestion
		minetest.log("warning", "[3d_armor] Registered armor "..name..
			" does not have \"_{material name}\" specified at the end of the item registration name")
```

Reply:
Updated 
```
		minetest.log("warning:[3d_armor] Registered armor "..name..
		" does not have \"_material\" specified at the end of the item registration name")
```
re-tested and confirmed working
```
2021-01-03 17:47:38: [Main]: warning:[3d_armor] Registered armor 3d_armor:helmetwood does not have "_material" specified at the end of the item registration name
```

Reply:
Thank you. Just for clarification: I suggested this change so that log levels work. There's already a "warning" log level, thus it would not make sense to log this message when warnings are explicitly turned off by a configuration.